app/controllers/api/v1/events_controller.rb

```diff
@@ -15,6 +15,19 @@ def create
     end
   end
 
+  def update
+    @event = Event.find(params[:id])
+
+    if @event.update(event)
+      render
```

Comment:
Since you have to explicitly put a `render` here, also put the status code `:ok`